docs/ilm-setup.asciidoc

```diff
@@ -0,0 +1,189 @@
+== Manual index lifecycle management
+
+APM Server allows users to manually set up an index lifecylce managment (ILM) policy.
+This setup is made possible because of Elasticsearch's
+{ref}/using-policies-rollover.html#using-policies-rollover[index rollover] action.
+
+Follow the guide below to set up a custom ILM policy for span indices.
+You can repeat the actions for any other indices.
+
+. Set up the default index template
++
+If you haven't already, you'll need to set up the default index template.
+This is accomplished by running the <<setup-command,`setup --template`>> command.
++
+--
+[source,js]
+-----------------------
+./apm-server setup --template
+-----------------------
+// CONSOLE
+--
+
+. Create a policy for spans.
++
+Rollover actions are triggered when one or more criteria are met.
+In this example, we're creating a policy named `apm_span_policy`.
++
+This policy rolls data over to a new index after _1 day_, or _50gb_, whichever comes first.
+It also tells the old indexes to delete after _10 days_.
+All of these values can be customized to your specific needs.
++
```

Comment:
For symmetry with the other sections, we could point that this only needs to be done once, rather than as part of each version upgrade.